Meilisearch Cloud: Include documentation for custom user bindings (#2680)
meilisearch/documentation: learn/cloud/analytics.mdx

**The previous API URL will remain functional**, but requests targeting it will not send any data to the analytics interface. If you created any custom API keys using the previous URL, you will need to replace them.

<Capsule intent="danger" title="Analytics and multi-search queries">

>   Meilisearch Cloud analytics silently ignores requests to [the `/multi-search`
>   route](/reference/api/multi_search). This is a known bug and will be fixed in
>   a future Cloud update.

Reviewer: ```suggestion
Meilisearch Cloud analytics silently ignores requests to [the `/multi-search` route](/reference/api/multi_search). This is a known bug and will be fixed in a future Cloud update.
```

  }'
```

> <Capsule intent="tip" title="Be advised!">
>   Meilisearch will try to automatically associate analytic events with the most
>   recent search of the user who triggered them. If you want to manually bind a
>   event to a search, include the same `X-MS-USER-ID` header in your search and
>   event request.
> </Capsule>

Reviewer: ```suggestion
<Capsule intent="tip" title="Binding analytics events to a search">
Meilisearch automatically associates analytics events with the most recent search of the user who triggered them. To manually bind an analytics event to a specific search, include the same `X-MS-USER-ID` header in your search and event requests.
</Capsule>
```
